optuna/terminator/erroreval.py

```diff
@@ -62,13 +84,36 @@ def evaluate(
 
 @experimental_class("3.2.0")
 def report_cross_validation_scores(trial: Trial, scores: list[float]) -> None:
+    """A function to report cross-validation scores of a trial.
+
+    This function should be called after cross-validation step within the trial to report its
+    scores. The reported scores are used to evaluate the statistical error for termination
```

Comment:
```suggestion
    This function should be called within the objective function to report
    the cross-validation scores. The reported scores are used to evaluate the statistical error for termination
```

Reply:
Just meant to be a little more easy to understand.

Reply:
Thanks! I applied the change